evaluator/diagnostic_converter/src/converter_node.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "converter_node.hpp"
+
+namespace diagnostic_converter
+{
+DiagnosticConverter::DiagnosticConverter(const rclcpp::NodeOptions & node_options)
+: Node("diagnostic_converter", node_options)
+{
+  using std::placeholders::_1;
+
+  size_t sub_counter = 0;
+  std::vector<std::string> diagnostic_topics = {"/diagnostic/planning_evaluator/metrics"};
```

Comment:
The list of diagnostic topics should be a parameter `diagnostic_topics`.

Reply:
You mean like this?
```
  declare_parameter<std::vector<std::string>>("diagnostic_topics", std::vector<std::string>());
  get_parameter<std::vector<std::string>>("diagnostic_topics", diagnostic_topics);
```